content/courses/solana-pay/solana-pay.md

```diff
@@ -210,11 +208,11 @@ export default async function handler(
 ) {
   if (req.method === "GET") {
```

Comment:
Please fix request and response everywhere. Use find and replace.

Reply:
All of them are fixed.